### ISSUE TITLE
Setting device_class attribute for some sensors

### ISSUE THREAD

**Issue by kaizersoje** [mawinkler/astroweather#9]
Is it possible to set the `device_class` for some sensors? I know this can be done in the customize section but I think it is better if they are set during creation.

| entity | device_class | 
| ---- | :-----: |
| `sensor.astroweather_2m_relative_humidity` | `humidity` | 
| `sensor.astroweather_2m_temperature` | `temperature` | 
| `sensor.astroweather_moon_next_rising` | `timestamp` | 
| `sensor.astroweather_moon_next_setting` | `timestamp` | 
| `sensor.astroweather_sun_next_rising` | `timestamp` | 
| `sensor.astroweather_sun_next_setting` | `timestamp` | 
| `sensor.astroweather_sun_next_rising_astronomical` | `timestamp` | 
| `sensor.astroweather_sun_next_setting_astronomical` | `timestamp` | 
| `sensor.astroweather_timestamp` | `timestamp` | 

I guess the one issue I can see is that the timestamp entities may not be in the correct ISO8601 format.

**Comment by mawinkler** [comment]
Hi, you're absolutely correct. I guess I missed that for the reason, that most sensors do not have a fitting `device_class`. For the ones with a match, I will include them in the next version.

**Comment by tomlut** [comment]
There's an issue with the lifted index device class.

```
Logger: homeassistant.components.sensor.recorder
Source: components/sensor/recorder.py:259
Integration: Sensor (documentation, issues)
First occurred: 12:30:10 (1 occurrences)
Last logged: 12:30:10

sensor.astroweather_lifted_index has unit ° which is unsupported for device_class temperature
```

**Comment by mawinkler** [comment]
Both fixed in release [v0.20.6](https://github.com/mawinkler/astroweather/releases/tag/v0.20.6)
